### DISCUSSION TITLE
Is CustomPaint being considered?

### DISCUSSION THREAD

**Discussion by lwanger** [flet-dev/flet#255] [Ideas]
I have need to display an org chart in a flet app (i.e. a graph structure with boxes and arrows). Looking online it looks like I need the equivalent of a Flutter CustomPaint widget. Is support for CustomPaint being considered? If not, is there an alternative I could use?

I know this probably opens up a lot of other areas that would need to be added, such as mouse events and gestures, but it seems like a big limitation to not be able to do any canvas drawing like functionality.

**Comment by FeodorFitsner** [comment]
Thanks for asking! Please vote/subscribe to this issue: https://github.com/flet-dev/flet/issues/205
